testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java

```diff
@@ -211,7 +224,7 @@ public void testFromUtcTimestamp()
                 .containsOnly(row(
                         "1969-12-31 16:00:00.123",
                         "1969-12-31 16:00:10.123",
-                        "1969-12-11 22:57:12.827",
+                        "1970-01-03 16:00:00.123",
```

Comment:
Would be easier to understand the motivation for this change if it was a preparatory commit.